integration/test-integration/src/test/resources/testng-cc-with-apim.xml

```diff
@@ -83,12 +81,6 @@
             <class name="org.wso2.choreo.connect.tests.setup.withapim.CcShutdownExecutor"/>
         </classes>
     </test>
-    <test name="cc-specific-tests-with-apim" parallel="false">
-        <classes>
-            <class name="org.wso2.choreo.connect.tests.setup.withapim.CcStartupExecutorTwo"/>
-            <class name="org.wso2.choreo.connect.tests.testcases.withapim.APIKeyHeaderTestCase"/>
```

Comment:
This was added with a separate config.toml to check whether the internal key is properly identified even if it is called with a query param or header which says "apikey". I think we need this.

Reply:
thanks. fixed with https://github.com/wso2/product-microgateway/pull/2501/commits/bfd4a9a3442475d917b8d577820f91c50a473793